### PR TITLE
Globalize postgresql

### PR DESCRIPTION
**Pull request by jannyHou**
Connect to strongloop/loopback#2422

### CODE REVIEW

**Comment by jannyHou** [comment]
@slnode test please

**Comment by rmg** [comment]
@slnode test please

**Comment by jannyHou** [comment]
@Amir-61 @loay A small globalization pr :) PTAL 
Thanks.

File: lib/migration.js [hunk 1]
```diff
@@ -821,7 +824,7 @@ function mixinMigration(PostgreSQL) {
         return column && [column, 'ASC'];
       });
     } else {
-      throw Error('Index keys definition appears to be invalid: ', keys);
+      throw Error(g.f('Index keys definition appears to be invalid: %s', keys));
```
Comment (loay) [inline lib/migration.js]: I think it should be: {{keys}} or even {{ Index keys}}

File: lib/postgresql.js [hunk 2]
```diff
@@ -385,10 +387,10 @@ PostgreSQL.prototype.buildExpression = function(columnName, operator,
           [operatorValue]);
     case 'regexp':
       if (operatorValue.global)
-        console.warn('PostgreSQL regex syntax does not respect the `g` flag');
+        g.warn('PostgreSQL regex syntax does not respect the {{`g`}} flag');
```
Comment (loay) [inline lib/postgresql.js]: {{PostgresSQL}}

File: lib/postgresql.js [hunk 3]
```diff
@@ -385,10 +387,10 @@ PostgreSQL.prototype.buildExpression = function(columnName, operator,
...
 
       if (operatorValue.multiline)
-        console.warn('PostgreSQL regex syntax does not respect the `m` flag');
+        g.warn('PostgreSQL regex syntax does not respect the {{`m`}} flag');
```
Comment (loay) [inline lib/postgresql.js]: {{PostgreSQL}}

File: lib/postgresql.js [hunk 4]
```diff
@@ -499,7 +501,7 @@ PostgreSQL.prototype.toColumnValue = function(prop, val) {
  * @returns {String} The place holder
  */
 PostgreSQL.prototype.getPlaceholderForIdentifier = function(key) {
-  throw new Error('Placeholder for identifiers is not supported');
+  throw new Error(g.f('Placeholder for identifiers is not supported'));
```
Comment (loay) [inline lib/postgresql.js]: {{Placeholder}}

**Comment by jannyHou** [comment]
@loay Thank you applied 👍

**Comment by loay** [comment]
LGTM for code. 
PTAL at CI failures.

**Comment by jannyHou** [comment]
@slnode test please

**Comment by jannyHou** [comment]
CI on stable platforms are all green: http://ci.strongloop.com/job/loopback-connector-postgresql/1032/

nvm platforms are failing due to `role jenkins does not exist`. Should be a config problem.
```
1) Autocreate schema if not exists "before all" hook:
     Uncaught error: role "jenkins" does not exist
      at Connection.parseE (node_modules/pg/lib/connection.js:551:11)
      at Connection.parseMessage (node_modules/pg/lib/connection.js:378:17)
      at Socket.<anonymous> (node_modules/pg/lib/connection.js:117:22)
      at Socket.<anonymous> (_stream_readable.js:765:14)
      at emitReadable_ (_stream_readable.js:427:10)
      at emitReadable (_stream_readable.js:423:5)
      at readableAddChunk (_stream_readable.js:166:9)
      at Socket.Readable.push (_stream_readable.js:128:10)
      at TCP.onread (net.js:529:21)

  2) Autocreate schema if not exists Uncaught error outside test suite:
     Uncaught TypeError: Cannot call method 'currentRetry' of undefined
  

  3) Autocreate schema if not exists Uncaught error outside test suite:
     Uncaught error: role "jenkins" does not exist
      at Connection.parseE (node_modules/pg/lib/connection.js:551:11)
      at Connection.parseMessage (node_modules/pg/lib/connection.js:378:17)
      at Socket.<anonymous> (node_modules/pg/lib/connection.js:117:22)
      at Socket.<anonymous> (_stream_readable.js:765:14)
      at emitReadable_ (_stream_readable.js:427:10)
      at emitReadable (_stream_readable.js:423:5)
      at readableAddChunk (_stream_readable.js:166:9)
      at Socket.Readable.push (_stream_readable.js:128:10)
      at TCP.onread (net.js:529:21)

```

**Pull request merged**